modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,177 @@
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;base64,<encoded_base64_string> <1>
+        path: /etc/NetworkManager/system-connections/bond0.nmconnection
+        filesystem: root
+        mode: 0600
+      - contents:
+# ...
+----
+<1> The Base64 encoded string that you have generated in the previous step.
++
+.. Create a connection profile for the bonded secondary interface and save the file in the `/etc/NetworkManager/system-connections/secondary-iface.nmconnection` path. For example, create the `bond1.nmconnection` file in your local directory with the following content:
++
+[source,ini]
+----
+[connection]
+id=eno2
+type=ethernet
+interface-name=eno2
+master=bond0
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+
+[ethernet]
+mtu=9000
+----
+
+. Create an Open vSwitch (OVS) bridge, `br-ex1`, on nodes by using OVN-Kubernetes network plugin to create an `extra_bridge` file`. Ensure that you save the file in the `/etc/ovnk/extra_bridge` path of the host. The file must state the interface name that supports the additional bridge and not the default interface that supports `br-ex`, which holds the primary IP address of the node. 
++
+.Example configuration for the `extra_bridge` file, `/etc/ovnk/extra_bridge`, that references a secondary interface that is attached to a VLAN
+[source,text]
+----
+ens4f1.1001
+----
+
+. Prepare the environment variables. Replace `<machine_role>` with the node role, such as `worker`, and replace `<interface_name>` with the name of your additional `br-ex` bridge name.
++
+[source,terminal]
+----
+$ export ROLE=<machine_role>
+----
++
+[source,terminal]
+----
+$ export INTERFACE=<interface_name>
+----
+
+. Create a machine config manifest file for configuring the network plugin by entering the following command in your terminal:
++
+[source,terminal]
+----
+$ oc create -f sec-bridge-cni.yaml
+----
+
+. Edit the machine config YAML file to have a configuration analogous to the following example:
++
+[source,yaml]
+----
+# ...
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: ${ROLE}
+  name: 12-${ROLE}-sec-bridge-cni
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - path: /etc/ovnk/extra_bridge
+          mode: 0420
+          overwrite: true
+          contents:
+            source: data:text/plain;charset=utf-8,${INTERFACE}
+          filesystem: root
+# ...
+----
+
+// Map nodes in your cluster to use the additional bridge by configuring the `/var/lib/ovnk/iface_default_hint` file.
+. Bind `br-ex1` to the secondary interface by configuring the `/var/lib/ovnk/iface_default_hint` file. With this file, you can change the priority of what bridge gets selected for your nodes.  
++
+.Example `iface_default_hint` file that states an additional bridge named `ens2f0.4085`
+[source,text]
+----
+DEFAULT=ens1f0
```

Comment:
variables are undefined here, content of file is simply: `<iface-name>` (example: `bond1`) or null

Reply:
:heavy_check_mark: